src/list-data/request-action-mapping.ts

```diff
@@ -61,32 +61,31 @@ export const BcMapping: RequestActionMappingI = {
 
 export const XproMapping: RequestActionMappingI = {
   AML: [
-    EntityTypes.XCR,
-    EntityTypes.XCP
+    EntityTypes.XCP, // maps to Corp Type "XCP"
+    EntityTypes.XCR // maps to Corp Type "XCR"
   ],
   ASSUMED: [
-    EntityTypes.XCR,
-    EntityTypes.RLC
+    EntityTypes.XCR, // maps to Corp Type "XCR"
+    EntityTypes.RLC // maps to Corp Type "LLC"
   ],
   REN: [
-    EntityTypes.XCR,
-    EntityTypes.XCP,
-    EntityTypes.RLC
+    EntityTypes.RLC, // maps to Corp Type "LLC"
+    EntityTypes.XCP, // maps to Corp Type "XCP"
+    EntityTypes.XCR // maps to Corp Type "XCR"
   ],
   REH: [
-    EntityTypes.A,
-    EntityTypes.LLC,
-    EntityTypes.XCP,
-    EntityTypes.XSO
+    EntityTypes.RLC, // maps to Corp Type "LLC"
+    EntityTypes.XCP, // maps to Corp Type "XCP"
+    EntityTypes.XSO, // maps to Corp Type "XS"
+    EntityTypes.XUL // maps to Corp Type "A"
   ],
   CHG: [
-    EntityTypes.A,
-    EntityTypes.LLC,
-    EntityTypes.XCP,
-    EntityTypes.XL,
-    EntityTypes.XLL,
-    EntityTypes.XLP,
-    EntityTypes.XP
```

Comment:
The bug ticket referred to name "722380 ALBERTA LTD.", which has corp type "A", so the list above needs to include "XUL". I have tested the new code and it's fine for this example.

Testing should also be done for:
- restorations for businesses of type "A", "LLC" and "XS"
- name changes for businesses of type "LLC", "XL", "XP" and "XS"